um_canonicalize/tags.yaml

```diff
@@ -79,9 +84,28 @@ categories:
     - INS
     - COM
     - VOC
+    - AT+ESS
+    - AT+ESS+ALL
+    - IN+ESS
+    - IN+ALL
+    - IN+ABL
+    - AT+ALL
+    - AT+ABL
+    - ON+ESS
+    - ON+ALL
+    - ON+ABL
+    - VOC+GEN
+    - NOM+ACC
+    - non{NOM}
```

Comment:
Can we give this tag a more standard-looking name? Is it supposed to be interpreted as "non-nominative"? In traditional grammar such cases are sometiems called "oblique": perhaps we can use that? Or alternatively, `NNOM`.

Reply:
This is intentional; it's the example that JCS used to introduce the `non{X}` modifier. Better, though, would be for the canonicalizer to handle these.